src/InstallTool.php

```diff
@@ -193,7 +193,7 @@ public function hasOldDatabase()
 
         $column = $this->connection->fetchAssoc($sql." AND COLUMN_NAME = 'sections'");
 
-        return 'varchar(1022)' !== $column['Type'];
+        return 'blob NULL' !== $column['Type'];
```

Comment:
Hm, but that wouldn't be completely right, would it? This function is supposed to check whether the database can be upgraded to Contao 4. Thus it must check for distinctive properties of the database fields that reliably indicate a recent enough database.